cacheobject/object.go

```diff
@@ -109,10 +93,29 @@ func CachableObject(obj *Object, rv *ObjectResults) {
 	if obj.ReqDirectives != nil && obj.ReqDirectives.NoStore {
 		rv.OutReasons = append(rv.OutReasons, ReasonRequestNoStore)
 	}
+}
+
+// LOW LEVEL API: Check if a response is cacheable.
+// This function doesn't reset the passed ObjectResults.
+func CachableResponseObject(obj *Object, rv *ObjectResults) {
```

Comment:
```suggestion
func CacheableResponseObject(obj *Object, rv *ObjectResults) {
```

Reply:
The spelling `Cachable` is used everywhere. See https://github.com/pquerna/cachecontrol/pull/17.

Reply:
IMO it should be consistent with the spelling. 
Then BC breaks can be temporised using deprecation